From 30fc7533e29c3e66ff63e98707d658b53747ce1b Mon Sep 17 00:00:00 2001
From: Rodriguez Fabien <fabienx.rodriguez@intel.com>
Date: Mon, 5 May 2014 17:41:12 +0200
Subject: [PATCH 312/429] isp: fix P-Unit time out at boot time

At boot time, PMU driver attemps to power down ISP whereas camera driver
is not enabled in edison kernel.
To avoid the time out at boot time, camera shall be considered as off.

Signed-off-by: Rodriguez Fabien <fabienx.rodriguez@intel.com>
---
 arch/x86/platform/intel-mid/intel_soc_pmu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/platform/intel-mid/intel_soc_pmu.c b/arch/x86/platform/intel-mid/intel_soc_pmu.c
index 6b1091c89f1d..255880c7a945 100644
--- a/arch/x86/platform/intel-mid/intel_soc_pmu.c
+++ b/arch/x86/platform/intel-mid/intel_soc_pmu.c
@@ -814,7 +814,7 @@ static bool update_nc_device_states(int i, pci_power_t state)
 void init_nc_device_states(void)
 {
 #if !IS_ENABLED(CONFIG_VIDEO_ATOMISP)
-	mid_pmu_cxt->camera_off = false;
+	mid_pmu_cxt->camera_off = true;
 #endif
 
 #ifndef GFX_ENABLE
-- 
2.37.3

